### ISSUE TITLE
[Bug] FastAPI generator outputs paths with an extra backslash /

### ISSUE THREAD

**Issue by ngoldack** [fern-api/fern#2710]
When using fernapi/fern-fastapi-server (version 0.4.4) it registers the paths with a wrong path in FastAPI.

`generators.yaml`:
```yaml
groups:
    server:
        generators:
            - name: fernapi/fern-fastapi-server
              version: 0.4.4
              output:
                  location: local-file-system
                  path: ../../apps/api_new/src/api/generated
              config:
                  include_validators: false
```

I am using following `openapi.yaml`:

```yaml openapi.yaml
...
paths:
    /health:
        get:
            summary: Health
            description: |
                "This endpoint is used to check the health of the API."
            operationId: getHealth
            tags:
                - health
            responses:
                "200":
                    description: OK
                    content:
                        application/json:
                            schema:
                                $ref: "#/components/schemas/Health"
                "400":
                    $ref: "#/components/responses/BadRequest"
                "500":
                    $ref: "#/components/responses/InternalServerError"
...
```

Generated AbstractService (error is near the bottom):
```py
...
@classmethod
    def __init_get_health(cls, router: fastapi.APIRouter) -> None:
        endpoint_function = inspect.signature(cls.get_health)
        new_parameters: typing.List[inspect.Parameter] = []
        for index, (parameter_name, parameter) in enumerate(endpoint_function.parameters.items()):
            if index == 0:
                new_parameters.append(parameter.replace(default=fastapi.Depends(cls)))
            else:
                new_parameters.append(parameter)
        setattr(cls.get_health, "__signature__", endpoint_function.replace(parameters=new_parameters))

        @functools.wraps(cls.get_health)
        def wrapper(*args: typing.Any, **kwargs: typing.Any) -> Health:
            try:
                return cls.get_health(*args, **kwargs)
            except (BadRequestError, InternalServerError) as e:
                raise e
            except FernHTTPException as e:
                logging.getLogger(f"{cls.__module__}.{cls.__name__}").warn(
                    f"Endpoint 'get_health' unexpectedly threw {e.__class__.__name__}. "
                    + f"If this was intentional, please add {e.__class__.__name__} to "
                    + "the endpoint's errors list in your Fern Definition."
                )
                raise e

        # this is necessary for FastAPI to find forward-ref'ed type hints.
        # https://github.com/tiangolo/fastapi/pull/5077
        wrapper.__globals__.update(cls.get_health.__globals__)

        router.get(
            path="//health", # <---
            response_model=Health,
            description=AbstractHealthService.get_health.__doc__,
            **get_route_args(cls.get_health, default_tag="health"),
        )(wrapper)
```

**Comment by armandobelardo** [comment]
Hi there @mitsosf and @ngoldack, would you be able to check in on this with the latest fern version (running `fern upgrade` as well as the [latest version of the FastAPI generator](https://github.com/fern-api/fern/blob/main/generators/python/fastapi/VERSION))

**Comment by Julius-11** [comment]
> Hi there @mitsosf and @ngoldack, would you be able to check in on this with the latest fern version (running `fern upgrade` as well as the [latest version of the FastAPI generator](https://github.com/fern-api/fern/blob/main/generators/python/fastapi/VERSION))

I ran into this same issue today using fast-api 0.9.1 and fern 0.29.0 running the [petstore](https://www.petstore-openapi.com/v3) example. All paths are generated with an extra leading backslash:

```Python
router.delete(
          path="//store/order/{order_id}",
          response_model=None,
          status_code=starlette.status.HTTP_204_NO_CONTENT,
          description=AbstractStoreService.delete_order.__doc__,
          **get_route_args(cls.delete_order, default_tag="store"),
)(wrapper)
```

**Comment by armandobelardo** [comment]
Thank you for confirming @Julius-11 we'll look into this this week!

**Comment by armandobelardo** [comment]
@Julius-11 this should be resolved in version `0.9.2` of fastapi generator, let me know if that works for you!

**Comment by Julius-11** [comment]
```Bash
 File "/src/fern/core/__init__.py", line 11, in <module> from .pydantic_utilities import deep_union_pydantic_dicts, pydantic_v1
ImportError: cannot import name 'deep_union_pydantic_dicts' from 'fern.core.pydantic_utilities' (/src/fern/core/pydantic_utilities.py)
```

After doing
```Bash
rm -r src/ fern/
fern init
fern add fern-fastapi-server
fern generate
```

And also after just doing a fern generator upgrade.

**Comment by armandobelardo** [comment]
This one's unrelated, but should be fixed in 0.9.3!
